torch/csrc/jit/passes/onnx.cpp

```diff
@@ -38,13 +38,126 @@ void removePrintOps(std::shared_ptr<Graph>& graph) {
   removePrintOps(graph->block());
 }
 
+void checkONNXCompatibility(const c10::FunctionSchema& schema) {
+  // in ONNX, all inputs are tensors, no support for tensor list
+  // so at most one input tensor list is supported
+  bool has_tensor_list = false;
+  const auto& args = schema.arguments();
+  for (const auto& arg : args) {
+    if (arg.name() == "_caffe2_preallocated_outputs") {
```

Comment:
don't duplicate the "_caffe2_preallocated_outputs" name, but include it from the header where it is defined

Reply:
yeah, good point, to unblock our folks, i will land this diff first and address it in the following diff.